apps/next/src/pages/AccountManagement/components/QRCode/QRCode.tsx

```diff
@@ -4,7 +4,7 @@ import QRCodeSVG from 'qrcode.react';
 import { FC } from 'react';
 import { useTranslation } from 'react-i18next';
 import { Redirect, useHistory } from 'react-router-dom';
-import { AddressItem } from '../AddressItem';
+import { AddressItem } from '../../../../components/Address/AddressItem';
```

Comment:
You can use the `@/` alias for shorter paths:  `@/components/Address/AddressItem`